<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>505</width>
    <height>406</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Регулярные выражения: поиск номеров телефонов</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="regExpEdit">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>321</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>(\d+-\d+-\d+)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Регулярное выражение:</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="sourceText">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>451</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="plainText">
     <string>Offsets:   12  sdgsd 14 99 231 7
213-12-14</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="resultText">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>451</width>
      <height>181</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QCheckBox" name="maxLenSearchBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>330</y>
      <width>221</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Жадный поиск строк</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>505</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="loadAction"/>
    <addaction name="saveAction"/>
    <addaction name="clearAction"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="loadAction">
   <property name="text">
    <string>Загрузка из файла</string>
   </property>
  </action>
  <action name="saveAction">
   <property name="text">
    <string>Сохранение в файл</string>
   </property>
  </action>
  <action name="clearAction">
   <property name="text">
    <string>Очистить текст</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>sourceText</sender>
   <signal>textChanged()</signal>
   <receiver>MainWindow</receiver>
   <slot>applyRegExp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>245</x>
     <y>118</y>
    </hint>
    <hint type="destinationlabel">
     <x>252</x>
     <y>185</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>regExpEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>applyRegExp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>310</x>
     <y>62</y>
    </hint>
    <hint type="destinationlabel">
     <x>252</x>
     <y>185</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>maxLenSearchBox</sender>
   <signal>stateChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>applyRegExp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>95</x>
     <y>371</y>
    </hint>
    <hint type="destinationlabel">
     <x>252</x>
     <y>202</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearAction</sender>
   <signal>triggered()</signal>
   <receiver>sourceText</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>245</x>
     <y>118</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saveAction</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>saveText()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>252</x>
     <y>202</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>loadAction</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>loadText()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>252</x>
     <y>202</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>applyRegExp()</slot>
  <slot>saveText()</slot>
  <slot>loadText()</slot>
 </slots>
</ui>
